cav_msgs/msg/BSMCoreData.msg

```diff
@@ -106,7 +106,7 @@ float32 STEER_WHEEL_ANGLE_MAX = 189.0
 float32 STEER_WHEEL_ANGLE_MIN = -189.0
 
 # Set of acceleration values in 3 orthogonal directions of the vehicle and with yaw rotation rates expressed as a structure.
-cav_msgs/AccelerationSet4Way accelSet
+cav_msgs/AccelerationSet4Way accelset
```

Comment:
What's the reasoning behind changing all these field names to remove the camel casing?

Reply:
In ROS2 the field names must be lowercase alphanumeric characters with underscores for separating words. otherwise it throws an [error](https://docs.ros.org/en/foxy/Concepts/About-ROS-Interfaces.html).

Reply:
@pmusau17 In that case, I would prefer we replace the camel case word separation with an underscore to keep the variable name readable. So this, for example, would become accel_set